codebuild_specs/amplify_general_config_tests.yml

```diff
@@ -10,3 +10,6 @@ phases:
   build:
     commands:
       - source ./shared-scripts.sh && _amplifyGeneralConfigTests
+artifacts:
+  files:
+    - 'shared-scripts.sh'
```

Comment:
what is this for ?

Reply:
codebuild requires a job to output an artifact, otherwise other jobs cannot depend on it downstream

Reply:
:-)